scenarioo-server/src/main/java/org/scenarioo/rest/search/SearchResource.java

```diff
@@ -38,14 +31,15 @@
 	@Produces("application/json")
 	@Path("/branch/{branchName}/build/{buildName}/search/{q}")
 	public SearchResponse search(@PathParam("branchName") final String branchName,
-												  @PathParam("buildName") final String buildName, @PathParam("q") final String q) {
+								 @PathParam("buildName") final String buildName, @PathParam("q") final String q,
+								 @QueryParam("includeHtml") boolean includeHtml) {
```

Comment:
Let's do this when we need it. It's probably not so easy to do this as it's a query parameter at the moment.